tests/tests_pytorch/loggers/test_neptune.py

```diff
@@ -40,7 +40,23 @@ def create_neptune_mock():
     Mostly due to fact, that windows tests were failing with MagicMock based strings, which were used to create local
     directories in FS.
     """
-    return MagicMock(init=MagicMock(return_value=MagicMock(__getitem__=MagicMock(side_effect=fetchable_paths))))
+    return MagicMock(
```

Comment:
We can merge both functions (`create_neptune_mock` and `create_neptune_offline_mock`) into one. In order to achieve it you'll need to prepare additional side_effect function for `init` that will return
```python
MagicMock(
     __getitem__=MagicMock(side_effect=fetchable_paths), exists=MagicMock(return_value=True)
)
```

based on `mode` argument (`mode=offline` vs non-existence). Then we'll be able to get back to one patch for all of the TestCase class.

Reply:
Done ✅